apps/ledger-live-mobile/scripts/e2e-ci.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
You need to add `set -e` at the beginning of the file.

Right now, if one of the command inside the build/test function encounters an error, the script will continue and will return 0. So it's impossible to catch an error from the CI side.

With `set -e` if a command encounters an error, the script will halt and will return the matching error code.